src/ws/sub_structs.rs

```diff
@@ -51,3 +51,18 @@ pub struct TradeInfo {
 pub struct UserData {
     pub fills: Vec<TradeInfo>,
 }
+
+#[derive(Deserialize, Clone, Debug)]
+#[allow(non_snake_case)]
```

Comment:
Let's name the field names descriptively and use aliases for the abbreviations, e.g. `#[serde(rename = "T")] time: u64` 

Here is the serialization format from the backend
```
#[derive(Debug, Clone, Serialize, Deserialize)]
pub(crate) struct Candle {                                                                                                                                                                                                                              
    #[serde(rename = "t")]
    pub(crate) time_open: u64,                                                                                                                                                                                                                           
    #[serde(rename = "T")]
    pub(crate) time_close: u64,                                                                                                                                                                                                                          
    #[serde(rename = "s")]
    pub(crate) coin: String,                                                                                                                                                                                                                             
    #[serde(rename = "i")]
    pub(crate) candle_interval: CandleInterval,                                                                                                                                                                                                          
    #[serde(rename = "o")]
    pub(crate) open: String,
    #[serde(rename = "c")]
    pub(crate) close: String,
    #[serde(rename = "h")]
    pub(crate) high: String,
    #[serde(rename = "l")]
    pub(crate) low: String,
    #[serde(rename = "v")]
    pub(crate) vlm: String,
    #[serde(rename = "n")]
    pub(crate) num_trades: usize,
}
```